test/jira-operations-test.js

```diff
@@ -35,6 +39,31 @@ describe('JIRA Operations Tests', function() {
         });
     });
 
+    it('Find parent from story with Sub-task parent', () => {
+      return findParent(dummyJira.issues.LinkedStory1, dummyJira)
+        .then(parent => {
+          parent.fields.issuetype.name.should.eql('Sub-task');
+        });
+    });
+
+    it('Find parent from story with Initiative, Epic, and Sub-task parents', () => {
+      return findParent(dummyJira.issues.LinkedStory2, dummyJira).then( parent => {
+        parent.fields.issuetype.name.should.eql('Initiative');
+      });
+    });
+
+    it('Find parent from story with Epic and Sub-task parents', () => {
+      return findParent(dummyJira.issues.LinkedStory3, dummyJira)
+        .then(parent => {
+          parent.fields.issuetype.name.should.eql('Epic');
+        });
+    });
+
+    it('Find parent\'s parent of a Sub-task that is under a story with a link to a Dispatcher Sub-task', ()=> {
+      findParent(dummyJira.issues.LinkedSubtask1, dummyJira)
+        .then(subtaskParent => findParent(subtaskParent, dummyJira).then(storyParent => findParent(storyParent, dummyJira).should.eventually.have.deep.property('fields.issuetype.name', 'Dispatcher')));
```

Comment:
Let's newline this.
```js
findParent(args)
  .then(...stuff)
  .then(...moreStuff)
  .should.eventually.have.deep.property(...things)
```

Now this line isn't so long that it triggers scrolling and I can more easily see where the actual assertion is taking place.